frontend/language/src/nb.json

```
@@ -762,9 +766,11 @@
   "resourceadm.about_resource_translation_description": "Beskrivelse",
   "resourceadm.about_resource_translation_right_description": "Delgasjonstekst",
   "resourceadm.about_resource_translation_title": "Navn på tjenesten",
+  "resourceadm.about_resource_use_rrr_show_text": "Ressursen {{showText}} bruke ressurseierstyrt rettighetsregister.",
```

Is it maybe better to use a different name than ```showText``` as it sounds like a boolean? 
Maybe like this: 
```suggestion
  "resourceadm.about_resource_use_rrr_show_text": "Ressursen {{resource}} bruke ressurseierstyrt rettighetsregister.",
```